ViewLayer/ViewContainer.h

```diff
@@ -2,17 +2,21 @@
 
 #include <QScopedPointer>
 
-class I_DisplayDashboardUI;
+enum Mode { RACE, DISPLAY };
```

Comment:
Originally I was thinking of creating a namespace in the header file and encapsulate the enum there, but I didn't realize your last suggestion was an option. That's defs better though, good question :)